ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1015,3 +1015,52 @@ def test_expand(
         x=x[0],
         shape=shape,
     )
+
+
+@st.composite
+def _concat_from_sequence_helper(draw):
+    shape = draw(helpers.get_shape(min_num_dims=1))
+    num_arrays = draw(
+        st.shared(helpers.ints(min_value=1, max_value=3), key="num_arrays")
+    )
+    dtype = draw(st.sampled_from(draw(helpers.get_dtypes("valid"))))
+    arrays = []
+    dtypes = [dtype for _ in range(num_arrays)]
+    for _ in range(num_arrays):
+        array = draw(helpers.array_values(dtype=dtype, shape=shape))
+        arrays.append(np.asarray(array, dtype=dtype))
+
+    axis = draw(
+        helpers.get_axis(shape=st.shared(helpers.get_shape(), key="value_shape"))
+    )
+    return dtypes, arrays, axis
+
```

Comment:
This is the simplest working way to generate the arguments
```
@st.composite
def _concat_from_sequence_helper(draw):
    dtypes, arrays, shape = draw(helpers.dtype_and_values(
        available_dtypes=helpers.get_dtypes("valid"),
        num_arrays=helpers.ints(min_value=1, max_value=6),
        ret_shape=True,
    ))
    axis = draw(helpers.get_axis(shape=shape))
    return dtypes, arrays, axis
```
I ran it locally and it seems to work, but there are other bugs to solve.

Reply:
I ran the test again locally and found out that it is giving a recursion error in local because it is trying to concatenate and stack 0 dimensional arrays while testing different cases, which is not possible. How should I proceed from here.